host/4/bullseye/amd64/dotnet/dotnet8-isolated/dotnet-isolated.Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Build the runtime from source
+ARG HOST_VERSION=4.22.0
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest=false
+
+RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
+    git clone --branch v${HOST_VERSION} https://github.com/Azure/azure-functions-host /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
+    HOST_COMMIT=$(git rev-list -1 HEAD) && \
+    dotnet publish -v q /p:BuildNumber=$BUILD_NUMBER /p:CommitHash=$HOST_COMMIT src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj -c Release --output /azure-functions-host --runtime linux-x64 && \
+    mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
+    rm -rf /root/.local /root/.nuget /src
+
+RUN apt-get update && \
+    apt-get install -y gnupg wget unzip
+
+# Include ASP.NET Core shared framework from dotnet/aspnet image.
+FROM mcr.microsoft.com/dotnet/aspnet:8.0-preview AS aspnet8
+
+FROM mcr.microsoft.com/dotnet/runtime:8.0-preview
```

Comment:
Being that the Dotnet Isolated images have a unique base image they will not get any layer sharing benefits.  Therefore, we should use the ASPNET image as our base instead of RUNTIME.  It should make the final image more performant.

Reply:
Since Docker memory usage is simple its backwards

If Set 2 is a superset of set 1, docker can't really union them well so we ended up with 
SET 1 [SET2 (includes overlap of set1] >>> Size than Set2

Even though the content should be the same.  Its due to the union file system of docker.